### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
419 advance fee scam bulk nuke
Those scam website are a part of the 419 advance fee scams called “Bitcoin Revolution” and also some of them are imaginary exchanges with no liquidity.

If you open the site from France, it will give you a famous French person, like movie star or TV host who “got rich from bitcoin”.

There is a false sense of urgency to “invest”.

The goal is to find guidable users who want to get rich quick, based off the bogus “evidence” presented on the fake “bitcoin revolution” scam website.
They are baited to “invest.”

Their portfolio grows, until they try to withdraw, then all is lost.

Often scammers spam DMs in discord or twitter or telegram or other platforms and lure users into the bogus websites as if they won something.

Then in order to “withdraw” the bonus, users must first depost a “fee”.

This money is always lost forever, if you are gullible enough to believe that a random exchange registered 2 days ago in Russia will gift you Bitcoin or Ethereum.

Spoiler: they won’t!

tl;dr: You won FAKE money, but you must deposit REAL money for the so called FEEs.

This is how a spam message looks

```
🚀🚀🚀 Congratulations! 🚀🚀🚀

In case you got that notification
You are one of the winners of our charity draw

Due to high trading volume on our exchange, we want to announce the promotion in BTC, which was made by CryptoCratex with help of Kraken, we decided to attract new participants and help crypto society in this difficult time

💰 Your prize: 0.7 BTC
🔑 Your code : WIN111

1) Sign up to /cryptocratex[.]com
2) Go to exchange settings on the main page
3) Apply the promocode
4)Done, you can use your funds

⏳ You must withdraw received BTC in 7 days, otherwise, the application will be declined invalid.
🔴 Promocode can be used only once
If you have any questions, you can contact support
```

Here is a list of fake 419 advance fee scams which [I have personally shut down myself.](https://twitter.com/dubstard/status/1514837690465890310)
```
/acutetradingfx.online
/ak-traders.com
/alphaearning.com
/asotplatform.com
/atikotrade.com
/axiemining.com
/biggestlegaltradingsystem.co
/biggestminitrade.com
/billxoptions.com
/binarysystemtrading.com
/bitex-fxtrading.com
/bitminingtrading.co
/bitstarminin.co
/bitstocklivetrading.com
/briskminersfx.com
/capital-bittrade.online
/cmcmarketpro.com
/co-tradex.com
/coinindexmarket.com
/coinkin.org
/coinminers.info
/confxminitrade.co
/criptofaci.com
/cryptafarm.com
/cryptexprofit.com
/crypto-booster.biz
/crypto-tokenrecovery.com
/cryptofin.biz
/cryptogain-invest.xyz
/cryptonitepips.com
/cryptotriplelive.com
/cryptotrustfx.live
/cryptoxtmining.com
/crytfxpro.com
/cwaptrade.com
/dailygrowthinvestment.com
/dailyincome247.com
/digitalcoinasset.ltd
/digitalcryptofx.com
/dxfinance.net
/expertreliable.co
/financial-trading.ltd
/fincapitalbank.com
/forex-capitaltrading.com
/fundsgaterecovery.net
/fxcapitalxchange.cf
/fxinvestmentbot.com
/glenholdings.com
/globalpower.ltd
/goldbinaryoptiontrading.com
/goldenbridgeassets.com
/goldenbtc.net
/greenhillstrade-fx.com
/hexatradefx.com
/hotinvestnow.com
/idbnk.online
/imperialassets.co
/innovationgoldfx.com
/inteldigital.trade
/investbanque.com
/kontcoins.com
/legalbtcmining.org
/linuscointrade.net
/lunotradingcompany.com
/mega-forexchange.ltd
/metamask.io-verification.com
/miningmarketpro.com
/mininings.com
/miracletradefx.com
/mullenfx.com
/net-wallets.com
/nexo-fx.com
/oandainvest.com
/octaglobalfx.com
/ofxglobaltrade.com
/optimalcointrade.com
/orion-tradings.com
/otc.eqibank.com
/otcex-investment.com
/paulgoldhunter.com
/paydragons.com
/pro-earners.com
/profitableoptionsfxinvestment.com
/profittextrade.com
/qantasfx.ltd
/revamp-traders.ltd
/robotmoney.net
/safebtcminings.com
/safecointrade.com
/safefxoption.com
/safefxoptions.com
/safemodetrading.com
/sbrokerforexpips.com
/securedverifiedoffice.com
/securefxtrade.org
/silverlinebank.org
/sinogoldlimited.com
/solariscapital.online
/standardcryptotraders.com
/stockmarketlabfx.net
/stockmartinc.com
/stockportfxtrade.com
/superfxtrading.co
/swiftlyxcapital.com
/tesla-exchange.ltd
/thegoldasset.org
/traderdesk.org
/tradestationfx247.com
/tradex-financial.com
/tradiine.com
/tronelakebank.com
/upsalefx.com
/vitexlimited.com
/safeedgetrade.com
```
gonna remove any dupes after the CI/CD auto parse does the heavy lifting for me.

### CODE REVIEW

**Review by dubstard — COMMENTED** [review]
Trim spaces
     +   "coinkin.org ",
    +   "cryptonitepips.com ",
    +   "revamp-traders.ltd ",
    +   "robotmoney.net ",
	
Remove dupes
	
	-   "asotplatform.com",
    -   "capital-bittrade.online",
    -   "crypto-tokenrecovery.com",
    -   "cryptoxtmining.com",
    -   "dailygrowthinvestment.com",
    -   "digitalcryptofx.com",
    -   "fundsgaterecovery.net",
    -   "kontcoins.com",
    -   "legalbtcmining.org",
    -   "linuscointrade.net",
    -   "mininings.com",
    -   "oandainvest.com",
    -   "ofxglobaltrade.com",
    -   "optimalcointrade.com",
    -   "paulgoldhunter.com",
    -   "safebtcminings.com",
    -   "safecointrade.com",
    -   "safemodetrading.com",
    -   "securefxtrade.org",
    -   "tradex-financial.com",

Fix 
	
"io-verification.com",

**Review by thwjanssen89 — APPROVED** [review]
